workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerHeartbeat/SendPlayerHeartbeatRequestSystem.cs

```diff
@@ -23,11 +23,10 @@ public class SendPlayerHeartbeatRequestSystem : ComponentSystem
         private struct Data
         {
             public readonly int Length;
-            [ReadOnly] public ComponentDataArray<PlayerHeartbeatClient.CommandSenders.PlayerHeartbeat> RequestSenders;
+            public ComponentDataArray<PlayerHeartbeatClient.CommandSenders.PlayerHeartbeat> RequestSenders;
```

Comment:
Missed a `[Inject]` API 😛

Reply:
will do this one in the next commit after PR feedback commit